### PR TITLE
Remove outdated note and unused include

### PR DESCRIPTION
**Pull request by tschaub**
Reading the latest published version of [the draft](https://portal.ogc.org/files/?artifact_id=101406&version=1), under Recommendation 7 on page 40, there are two notes that look like this:

image

The second note is confusing in that it refers to a `...` that doesn't appear in a previous note.

It looks like f4c26fcc4c469fdf079464e5a581c03cdb2607a7 removed the note that this was referring to: https://github.com/opengeospatial/ogcapi-tiles/blob/91a36c89746d1c6f29cf18e57f646cadef1e33cf/core/standard/clause_7_tile_core.adoc#L110

Later, 177600a1b739ccf671d0bd31cfb7e19091b66e89 commented out an `include::requirements/tileset/REQ_tile-template.adoc` and the file that it referred to was then removed in 045eebacc7be01db15347e719dc070cbc610b16e.

This branch proposes removing the outdated note and the commented out `include` directive.

### CODE REVIEW

**Review by jerstlouis — APPROVED** [review]

**Review by jerstlouis — COMMENTED** [review]
We could potentially leave a note that says something like:

> NOTE: When this conformance class is used together with the GeoData Tilesets conformance class, the geospatial data resource URL is typically the first part of the URL template, but this standard does not mandate this (e.g., the tileset metadata and tiles could be hosted on more affordable object storage).

**Comment by tschaub** [comment]
I can update the commit to add that note if that is preferable.

**Comment by ghobona** [comment]
@joanma747 @jerstlouis I'm in the process of converting the document to metanorma asciidoc. Please review this PR, approve it, but do not merge it. I will merge the PR after completing the conversion to metanorma asciidoc (in mid September 2022).

**Comment by ghobona** [comment]
Closing this PR without merging because the requested changes have been implemented:

https://github.com/opengeospatial/ogcapi-tiles/commit/fc4b2815f0e99bfcb3bdc727224eb8a88c51302e

https://github.com/opengeospatial/ogcapi-tiles/commit/e24549a5013a45f3628ab25886de3d0654e110ef

**Pull request closed without merging**
